zapisy/apps/grade/poll/utils.py

```diff
@@ -84,12 +87,20 @@ def group(entries: List[Poll], sort=False) -> dict:
 
     for entry in entries:
         if entry is not None:
+            is_own = employee in [entry.owner, entry.teacher, entry.gcowner]
             category = entry.category
             subcategory = entry.subcategory
+            href = reverse('grade-poll-results',
+                           kwargs={'semester_id': semester_id, 'poll_id': entry.id})
+            entrydict = entry.to_dict()
+            entrydict.update({
+                'number_of_submissions': entry.number_of_submissions,
+                'is_own': is_own,
+                'href': href})
             if subcategory not in grouped_entries[category]:
                 if entry.semester:  # whether the entry is a general poll
-                    output[category].append(entry)
-                grouped_entries[category].append(entry)
+                    output[category].append(entrydict)
```

Comment:
```suggestion
                    entrydict[is_own] = True
                    output[category].append(entrydict)
```
i po tej zmianie będzie należało wywalić specjalne traktowanie ankiet ogólnych z pliku `.vue`.

Reply:
(Ponoć to nie działa – no to należałoby wymyślić coś innego, co pozwoli uprościć logikę frontendu)